app/models/organization.rb

```diff
@@ -254,4 +255,18 @@ def generate_access_token
       self.access_token = SecureRandom.hex
     end while self.class.exists?(access_token: access_token)
   end
+
+  private
+
+  def set_ambassador_organization_defaults
+    return unless kind == "ambassador"
+    self.show_on_map = false
+    self.lock_show_on_map = false
+    self.api_access_approved = false
+    self.approved = true
+    self.website = nil
+    self.ascend_name = nil
+    self.parent_organization_id = nil
+    self.locations = locations.reload.limit(1) if locations.count > 1
```

Comment:
I'm scared of this. 

I can see a scenario where we decide we need multiple locations and try to add them, and it takes a while to figure out why they aren't saving (or why a new one is replacing another one, or whatever happens).

I realize I told you to do this initially, but I think figuring out a way to lock it on the frontend will be good enough. And actually, you can just ignore the locations entirely because it's not clear what we're going to do with the locations.